bpf/lib/nodeport.h

```diff
@@ -1035,6 +1006,9 @@ static __always_inline int rev_nodeport_lb6(struct __ctx_buff *ctx, __u32 *ifind
 	l4_off = l3_off + hdrlen;
 	csum_l4_offset_and_flags(tuple.nexthdr, &csum_off);
 
+	if (nat_46x64_fib)
+		goto skip_rev_dnat;
+
```

Comment:
If you do a quick `s/tuple/ip6` cleanup in the FIB section, this check could go at the very top of the function. Just to emphasize that we really just pass through here to pick up the FIB.